src/pipeline/training_pipeline.py

```diff
@@ -43,16 +43,10 @@ def _setup_gan_loss_functions(self):
 
     def _setup_lr_schedulers(self):
         self.lr_schedulers = {}
-        for network_name, optimizer in self.optimizers.items():
-            lr_scheduler_name = f"lr_scheduler_{network_name}"
-            if lr_scheduler_name not in global_config:
-                logger.warning(
-                    f"{lr_scheduler_name} not in global_config; using default lr_scheduler for {network_name}"
-                )
-                lr_scheduler_name = 'lr_scheduler'
-            self.lr_schedulers[network_name] = get_instance(
-                torch.optim.lr_scheduler, lr_scheduler_name, global_config, optimizer
-            )
+        for optimizer_name, optimizer in self.optimizers.items():
+            entry = global_config['lr_schedulers'][optimizer_name]
```

Comment:
Same here, we may need to keep the warning message

Reply:
The names of `lr_schedulers` and `optimizers` should be strictly the same, therefore the warning messages won't be used.